CHANGELOG.md

```diff
@@ -2,7 +2,6 @@
 
 * Do not use computeIfAbsent in ScopeFactory in order to avoid ConcurrentModificationException in the IntelliJ plugin
 * Allow graph processing to continue even when a ParsingError is encountered. A malformed Scope will simply not be included in the ResolvedGraph.
-* Handle implicit Kotlin Nullability annotations.
```

Comment:
Not including IntelliJ changes in the CHANGELOG for now since it isn't yet in a working state.